lib/TBDGen/TBDGen.cpp

```diff
@@ -43,16 +43,17 @@ static bool isGlobalOrStaticVar(VarDecl *VD) {
 
 void TBDGenVisitor::visitPatternBindingDecl(PatternBindingDecl *PBD) {
   for (auto &entry : PBD->getPatternList()) {
-    auto *var = entry.getAnchoringVarDecl();
-
-    // Non-global variables might have an explicit initializer symbol.
-    if (entry.getNonLazyInit() && !isGlobalOrStaticVar(var)) {
-      auto declRef =
-          SILDeclRef(var, SILDeclRef::Kind::StoredPropertyInitializer);
-      // Stored property initializers for public properties are currently
-      // public.
-      addSymbol(declRef);
-    }
+    entry.getPattern()->forEachVariable([&](VarDecl *VD) {
+      // Non-global variables might have an explicit initializer symbol.
+      if (VD->getAttrs().hasAttribute<HasInitialValueAttr>() &&
+          !isGlobalOrStaticVar(VD)) {
```

Comment:
Oh no, I forgot that this is also relevant for classes. \*sigh\* Seems reasonable. Is it worth checking to see if this is a resilient module, though? (ModuleDecl::getResilienceStrategy)

Reply:
Yes, might as well do that change here.